sections/semantics-forms.include

```diff
@@ -9978,6 +9978,11 @@ out of 233 257 824 bytes available&lt;/meter&gt;&lt;/p&gt;
   author to indicate that a control is to be focused as soon as the page is loaded or as soon as the
   <code>dialog</code> within which it finds itself is shown, allowing the user to just start typing
   without having to manually focus the main control.
+
+  <div class="note">
+  When navigating to a fragment of a page, such as foo.bar#fragment, the fragment identifier takes precedence and the <{formelements/autofocus}> attribute has no effect.
```

Comment:
should probably make foo.bar https://foo.bar (and I am not sure if there is a .bar TLD although there may well be. perhaps example.com would be better).

Other than that, this looks exactly right - feel free to merge directly if you change the URL.